ts/features/itwallet/presentation/remote/screens/ItwRemoteClaimsDisclosureScreen.tsx

```diff
@@ -73,55 +92,161 @@ const ItwRemoteClaimsDisclosureScreen = (params: ScreenProps) => {
-            "features.itWallet.issuance.credentialAuth.disclaimer.0"
+        <View style={styles.claimsSelection}>
+          <ButtonLink
+            label={I18n.t(
+              `global.buttons.${
+                allOptionalClaimsSelected ? "deselectAll" : "selectAll"
+              }`
+            )}
+            onPress={toggleAllOptionalClaims}
+          />
+        </View>
+      </View>
+      <ItwOptionalClaimsList
+        items={ItwRemotePresentationClaimsMock.optional}
+        selectedClaims={selectedOptionalClaims}
+        onSelectionChange={toggleOptionalClaims}
+      />
+      {!allOptionalClaimsSelected && (
+        <Animated.View
+          entering={FadeIn.duration(200)}
+          exiting={FadeOut.duration(200)}
+          layout={LinearTransition.duration(200)}
+        >
+          <Alert
+            variant="info"
+            content={I18n.t(
+              "features.itWallet.presentation.selectiveDisclosure.optionalClaimsAlert"
+            )}
+          />
+        </Animated.View>
+      )}
+    </VStack>
+  );
+
+  return (
+    <ForceScrollDownView>
+      <ContentWrapper>
+        <VStack space={24}>
+          <ItwDataExchangeIcons
+            requesterLogoUri={require("../../../../../../img/features/itWallet/issuer/IPZS.png")} // TODO: get the Relying Party logo
+          />
+          <H2>
+            {I18n.t("features.itWallet.presentation.selectiveDisclosure.title")}
+          </H2>
+        </VStack>
+        <VSpacer size={16} />
+        <IOMarkdown
+          content={I18n.t(
+            "features.itWallet.presentation.selectiveDisclosure.subtitle",
+            { relyingParty: RP_MOCK_NAME }
           )}
         />
         <VSpacer size={24} />
-        <FeatureInfo
-          iconName="trashcan"
-          body={I18n.t(
-            "features.itWallet.issuance.credentialAuth.disclaimer.1"
-          )}
-        />
+
+        <VStack space={24}>
+          <View>
+            <ListItemHeader
+              label={I18n.t(
+                "features.itWallet.presentation.selectiveDisclosure.requiredClaims"
+              )}
+              iconName="security"
+              iconColor="grey-700"
+            />
+            <ItwMemoizedRequiredClaimsList
+              items={ItwRemotePresentationClaimsMock.required}
+            />
+          </View>
+
+          {renderOptionalClaims()}
+
+          <FeatureInfo
+            iconName="fornitori"
+            body={I18n.t(
+              "features.itWallet.presentation.selectiveDisclosure.disclaimer.0"
+            )}
+          />
+          <FeatureInfo
+            iconName="trashcan"
+            body={I18n.t(
+              "features.itWallet.presentation.selectiveDisclosure.disclaimer.1"
+            )}
+          />
+          <IOMarkdown
+            content={I18n.t(
+              "features.itWallet.presentation.selectiveDisclosure.tos",
+              { privacyUrl: RP_MOCK_PRIVACY_URL }
+            )}
+          />
```

Comment:
What do you think about wrapping with an `Animated.View` the info below the `Alert` too? That will make the animation smoother!

```suggestion
          <Animated.View
            entering={FadeIn.duration(200)}
            exiting={FadeOut.duration(200)}
            layout={LinearTransition.duration(200)}
          >
            <FeatureInfo
              iconName="fornitori"
              body={I18n.t(
                "features.itWallet.presentation.selectiveDisclosure.disclaimer.0"
              )}
            />
            <FeatureInfo
              iconName="trashcan"
              body={I18n.t(
                "features.itWallet.presentation.selectiveDisclosure.disclaimer.1"
              )}
            />
            <IOMarkdown
              content={I18n.t(
                "features.itWallet.presentation.selectiveDisclosure.tos",
                { privacyUrl: RP_MOCK_PRIVACY_URL }
              )}
            />
          </Animated.View>
```